[ 2025-03-24 16:07:39,697 ] 248 django.request - WARNING - Not Found: /profile/user/font.css
[ 2025-03-24 16:07:40,490 ] 248 django.request - WARNING - Not Found: /edit/font.css
[ 2025-03-24 16:07:45,742 ] 248 django.request - WARNING - Not Found: /profile/user/font.css
[ 2025-03-24 16:07:47,398 ] 248 django.request - WARNING - Not Found: /font.css
[ 2025-03-24 16:08:15,667 ] 248 django.request - WARNING - Not Found: /advertisers/add-advertisement/font.css
[ 2025-03-24 16:08:16,854 ] 248 django.request - WARNING - Not Found: /advertisers/profile/usera/font.css
[ 2025-03-24 16:08:18,236 ] 248 django.request - WARNING - Not Found: /advertisers/edit/font.css
[ 2025-03-24 16:08:29,382 ] 248 django.request - WARNING - Not Found: /profile/usera/
[ 2025-03-24 16:08:49,319 ] 248 django.request - ERROR - Internal Server Error: /posts/
Traceback (most recent call last):
  File "C:\Users\vivek\AppData\Local\Programs\Python\Python310\lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
  File "C:\Users\vivek\AppData\Local\Programs\Python\Python310\lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "C:\Users\vivek\AppData\Local\Programs\Python\Python310\lib\site-packages\django\views\decorators\cache.py", line 80, in _view_wrapper
    response = view_func(request, *args, **kwargs)
  File "C:\Users\vivek\AppData\Local\Programs\Python\Python310\lib\site-packages\django\contrib\auth\decorators.py", line 60, in _view_wrapper
    return view_func(request, *args, **kwargs)
  File "C:\Users\vivek\Desktop\Projects\Cyber Guardia\Cyber-Bullying-Detection\posts\views.py", line 30, in post_list
    user_interests = [interest.strip().lower().replace(" ", "") for interest in user_profile.area_of_interest.split(",")]
AttributeError: 'NoneType' object has no attribute 'area_of_interest'
[ 2025-03-24 16:09:01,943 ] 248 django.request - WARNING - Not Found: /advertisers/edit/font.css
[ 2025-03-24 16:09:04,869 ] 248 django.request - WARNING - Not Found: /profile/usera/
[ 2025-03-24 16:09:08,429 ] 248 django.request - ERROR - Internal Server Error: /posts/
Traceback (most recent call last):
  File "C:\Users\vivek\AppData\Local\Programs\Python\Python310\lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
  File "C:\Users\vivek\AppData\Local\Programs\Python\Python310\lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "C:\Users\vivek\AppData\Local\Programs\Python\Python310\lib\site-packages\django\views\decorators\cache.py", line 80, in _view_wrapper
    response = view_func(request, *args, **kwargs)
  File "C:\Users\vivek\AppData\Local\Programs\Python\Python310\lib\site-packages\django\contrib\auth\decorators.py", line 60, in _view_wrapper
    return view_func(request, *args, **kwargs)
  File "C:\Users\vivek\Desktop\Projects\Cyber Guardia\Cyber-Bullying-Detection\posts\views.py", line 30, in post_list
    user_interests = [interest.strip().lower().replace(" ", "") for interest in user_profile.area_of_interest.split(",")]
AttributeError: 'NoneType' object has no attribute 'area_of_interest'
[ 2025-03-24 16:09:15,070 ] 248 django.request - ERROR - Internal Server Error: /posts/
Traceback (most recent call last):
  File "C:\Users\vivek\AppData\Local\Programs\Python\Python310\lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
  File "C:\Users\vivek\AppData\Local\Programs\Python\Python310\lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "C:\Users\vivek\AppData\Local\Programs\Python\Python310\lib\site-packages\django\views\decorators\cache.py", line 80, in _view_wrapper
    response = view_func(request, *args, **kwargs)
  File "C:\Users\vivek\AppData\Local\Programs\Python\Python310\lib\site-packages\django\contrib\auth\decorators.py", line 60, in _view_wrapper
    return view_func(request, *args, **kwargs)
  File "C:\Users\vivek\Desktop\Projects\Cyber Guardia\Cyber-Bullying-Detection\posts\views.py", line 30, in post_list
    user_interests = [interest.strip().lower().replace(" ", "") for interest in user_profile.area_of_interest.split(",")]
AttributeError: 'NoneType' object has no attribute 'area_of_interest'
[ 2025-03-24 16:09:18,065 ] 248 django.request - WARNING - Not Found: /font.css
[ 2025-03-24 16:09:28,408 ] 248 django.request - WARNING - Not Found: /advertisers/add-advertisement/font.css
[ 2025-03-24 16:09:30,737 ] 248 django.request - WARNING - Not Found: /advertisers/profile/usera/font.css
[ 2025-03-24 16:10:34,801 ] 248 django.request - WARNING - Not Found: /advertisers/profile/usera/font.css
[ 2025-03-24 16:10:35,791 ] 248 django.request - WARNING - Not Found: /advertisers/edit/font.css
[ 2025-03-24 16:10:45,630 ] 248 django.request - WARNING - Not Found: /advertisers/profile/usera/font.css
[ 2025-03-24 16:10:59,817 ] 248 django.request - WARNING - Not Found: /advertisers/profile/usera/font.css
[ 2025-03-24 16:30:45,029 ] 248 django.request - WARNING - Not Found: /advertisers/profile/usera/font.css
[ 2025-03-24 16:30:46,823 ] 248 django.request - WARNING - Not Found: /advertisers/profile/usera/font.css
[ 2025-03-24 16:30:49,618 ] 248 django.request - WARNING - Not Found: /advertisers/edit/font.css
[ 2025-03-24 16:30:54,139 ] 248 django.request - WARNING - Not Found: /advertisers/profile/usera/font.css
[ 2025-03-24 16:31:20,468 ] 248 django.request - WARNING - Not Found: /advertisers/profile/usera/font.css
[ 2025-03-24 16:31:22,136 ] 248 django.request - WARNING - Not Found: /advertisers/profile/usera/font.css
[ 2025-03-24 16:31:28,884 ] 248 django.request - WARNING - Not Found: /advertisers/profile/usera/font.css
[ 2025-03-24 16:32:09,448 ] 248 django.request - WARNING - Not Found: /advertisers/profile/usera/font.css
[ 2025-03-24 16:32:11,147 ] 248 django.request - WARNING - Not Found: /advertisers/profile/usera/font.css
[ 2025-03-24 16:33:01,231 ] 248 django.request - WARNING - Not Found: /advertisers/profile/usera/font.css
[ 2025-03-24 16:33:25,490 ] 248 django.request - WARNING - Not Found: /advertisers/profile/usera/font.css
[ 2025-03-24 16:33:29,635 ] 248 django.request - WARNING - Not Found: /advertisers/profile/usera/font.css
[ 2025-03-24 17:26:06,049 ] 248 django.request - WARNING - Not Found: /advertisers/edit/font.css
[ 2025-03-24 17:26:08,446 ] 248 django.request - WARNING - Not Found: /advertisers/edit/font.css
[ 2025-03-24 17:26:22,159 ] 248 django.request - WARNING - Not Found: /advertisers/edit/font.css
[ 2025-03-24 17:26:23,324 ] 248 django.request - WARNING - Not Found: /advertisers/profile/usera/font.css
[ 2025-03-24 17:26:28,047 ] 248 django.request - WARNING - Not Found: /advertisers/profile/usera/font.css
[ 2025-03-24 17:27:00,238 ] 248 django.request - WARNING - Not Found: /advertisers/edit/font.css
[ 2025-03-24 17:27:02,313 ] 248 django.request - WARNING - Not Found: /advertisers/profile/usera/font.css
[ 2025-03-24 17:27:08,262 ] 248 django.request - WARNING - Not Found: /advertisers/profile/usera/font.css
[ 2025-03-24 17:27:37,597 ] 248 django.request - WARNING - Not Found: /advertisers/profile/usera/font.css
[ 2025-03-24 17:27:57,849 ] 248 django.request - WARNING - Not Found: /advertisers/profile/usera/font.css
[ 2025-03-24 17:27:58,714 ] 248 django.request - WARNING - Not Found: /advertisers/edit/font.css
[ 2025-03-24 17:28:00,454 ] 248 django.request - WARNING - Not Found: /advertisers/profile/usera/font.css
[ 2025-03-24 17:28:46,204 ] 248 django.request - WARNING - Not Found: /advertisers/profile/usera/font.css
[ 2025-03-24 17:28:54,236 ] 248 django.request - WARNING - Not Found: /advertisers/profile/usera/font.css
[ 2025-03-24 17:29:09,675 ] 248 django.request - WARNING - Not Found: /advertisers/edit/font.css
[ 2025-03-24 17:29:18,355 ] 248 django.request - WARNING - Not Found: /advertisers/profile/usera/font.css
[ 2025-03-24 17:29:37,750 ] 248 django.request - WARNING - Not Found: /advertisers/edit/font.css
[ 2025-03-24 17:29:43,258 ] 248 django.request - WARNING - Not Found: /advertisers/profile/usera/font.css
[ 2025-03-24 17:32:57,044 ] 248 django.request - WARNING - Not Found: /advertisers/profile/usera/font.css
[ 2025-03-24 17:32:58,083 ] 248 django.request - ERROR - Internal Server Error: /advertisers/edit/
Traceback (most recent call last):
  File "C:\Users\vivek\AppData\Local\Programs\Python\Python310\lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
  File "C:\Users\vivek\AppData\Local\Programs\Python\Python310\lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "C:\Users\vivek\AppData\Local\Programs\Python\Python310\lib\site-packages\django\contrib\auth\decorators.py", line 60, in _view_wrapper
    return view_func(request, *args, **kwargs)
  File "C:\Users\vivek\Desktop\Projects\Cyber Guardia\Cyber-Bullying-Detection\advertisers\views.py", line 100, in edit_advertiser_profile
    return render(request, 'advertisers/adver_edit_profile.html', {'form': form})
  File "C:\Users\vivek\AppData\Local\Programs\Python\Python310\lib\site-packages\django\shortcuts.py", line 25, in render
    content = loader.render_to_string(template_name, context, request, using=using)
  File "C:\Users\vivek\AppData\Local\Programs\Python\Python310\lib\site-packages\django\template\loader.py", line 61, in render_to_string
    template = get_template(template_name, using=using)
  File "C:\Users\vivek\AppData\Local\Programs\Python\Python310\lib\site-packages\django\template\loader.py", line 15, in get_template
    return engine.get_template(template_name)
  File "C:\Users\vivek\AppData\Local\Programs\Python\Python310\lib\site-packages\django\template\backends\django.py", line 79, in get_template
    return Template(self.engine.get_template(template_name), self)
  File "C:\Users\vivek\AppData\Local\Programs\Python\Python310\lib\site-packages\django\template\engine.py", line 177, in get_template
    template, origin = self.find_template(template_name)
  File "C:\Users\vivek\AppData\Local\Programs\Python\Python310\lib\site-packages\django\template\engine.py", line 159, in find_template
    template = loader.get_template(name, skip=skip)
  File "C:\Users\vivek\AppData\Local\Programs\Python\Python310\lib\site-packages\django\template\loaders\cached.py", line 57, in get_template
    template = super().get_template(template_name, skip)
  File "C:\Users\vivek\AppData\Local\Programs\Python\Python310\lib\site-packages\django\template\loaders\base.py", line 28, in get_template
    return Template(
  File "C:\Users\vivek\AppData\Local\Programs\Python\Python310\lib\site-packages\django\template\base.py", line 154, in __init__
    self.nodelist = self.compile_nodelist()
  File "C:\Users\vivek\AppData\Local\Programs\Python\Python310\lib\site-packages\django\template\base.py", line 196, in compile_nodelist
    nodelist = parser.parse()
  File "C:\Users\vivek\AppData\Local\Programs\Python\Python310\lib\site-packages\django\template\base.py", line 518, in parse
    raise self.error(token, e)
  File "C:\Users\vivek\AppData\Local\Programs\Python\Python310\lib\site-packages\django\template\base.py", line 516, in parse
    compiled_result = compile_func(self, token)
  File "C:\Users\vivek\AppData\Local\Programs\Python\Python310\lib\site-packages\django\template\loader_tags.py", line 295, in do_extends
    nodelist = parser.parse()
  File "C:\Users\vivek\AppData\Local\Programs\Python\Python310\lib\site-packages\django\template\base.py", line 518, in parse
    raise self.error(token, e)
  File "C:\Users\vivek\AppData\Local\Programs\Python\Python310\lib\site-packages\django\template\base.py", line 516, in parse
    compiled_result = compile_func(self, token)
  File "C:\Users\vivek\AppData\Local\Programs\Python\Python310\lib\site-packages\django\template\loader_tags.py", line 234, in do_block
    nodelist = parser.parse(("endblock",))
  File "C:\Users\vivek\AppData\Local\Programs\Python\Python310\lib\site-packages\django\template\base.py", line 489, in parse
    raise self.error(token, e)
  File "C:\Users\vivek\AppData\Local\Programs\Python\Python310\lib\site-packages\django\template\base.py", line 487, in parse
    filter_expression = self.compile_filter(token.contents)
  File "C:\Users\vivek\AppData\Local\Programs\Python\Python310\lib\site-packages\django\template\base.py", line 605, in compile_filter
    return FilterExpression(token, self)
  File "C:\Users\vivek\AppData\Local\Programs\Python\Python310\lib\site-packages\django\template\base.py", line 706, in __init__
    raise TemplateSyntaxError(
django.template.exceptions.TemplateSyntaxError: Could not parse the remainder: ' if profile.profile_image else '{% static 'images/profile-face.png' %}'' from 'profile.profile_image.url if profile.profile_image else '{% static 'images/profile-face.png' %}''
[ 2025-03-24 17:33:46,972 ] 248 django.request - ERROR - Internal Server Error: /advertisers/edit/
Traceback (most recent call last):
  File "C:\Users\vivek\AppData\Local\Programs\Python\Python310\lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
  File "C:\Users\vivek\AppData\Local\Programs\Python\Python310\lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "C:\Users\vivek\AppData\Local\Programs\Python\Python310\lib\site-packages\django\contrib\auth\decorators.py", line 60, in _view_wrapper
    return view_func(request, *args, **kwargs)
  File "C:\Users\vivek\Desktop\Projects\Cyber Guardia\Cyber-Bullying-Detection\advertisers\views.py", line 100, in edit_advertiser_profile
    return render(request, 'advertisers/adver_edit_profile.html', {'form': form})
  File "C:\Users\vivek\AppData\Local\Programs\Python\Python310\lib\site-packages\django\shortcuts.py", line 25, in render
    content = loader.render_to_string(template_name, context, request, using=using)
  File "C:\Users\vivek\AppData\Local\Programs\Python\Python310\lib\site-packages\django\template\loader.py", line 61, in render_to_string
    template = get_template(template_name, using=using)
  File "C:\Users\vivek\AppData\Local\Programs\Python\Python310\lib\site-packages\django\template\loader.py", line 15, in get_template
    return engine.get_template(template_name)
  File "C:\Users\vivek\AppData\Local\Programs\Python\Python310\lib\site-packages\django\template\backends\django.py", line 79, in get_template
    return Template(self.engine.get_template(template_name), self)
  File "C:\Users\vivek\AppData\Local\Programs\Python\Python310\lib\site-packages\django\template\engine.py", line 177, in get_template
    template, origin = self.find_template(template_name)
  File "C:\Users\vivek\AppData\Local\Programs\Python\Python310\lib\site-packages\django\template\engine.py", line 159, in find_template
    template = loader.get_template(name, skip=skip)
  File "C:\Users\vivek\AppData\Local\Programs\Python\Python310\lib\site-packages\django\template\loaders\cached.py", line 57, in get_template
    template = super().get_template(template_name, skip)
  File "C:\Users\vivek\AppData\Local\Programs\Python\Python310\lib\site-packages\django\template\loaders\base.py", line 28, in get_template
    return Template(
  File "C:\Users\vivek\AppData\Local\Programs\Python\Python310\lib\site-packages\django\template\base.py", line 154, in __init__
    self.nodelist = self.compile_nodelist()
  File "C:\Users\vivek\AppData\Local\Programs\Python\Python310\lib\site-packages\django\template\base.py", line 196, in compile_nodelist
    nodelist = parser.parse()
  File "C:\Users\vivek\AppData\Local\Programs\Python\Python310\lib\site-packages\django\template\base.py", line 518, in parse
    raise self.error(token, e)
  File "C:\Users\vivek\AppData\Local\Programs\Python\Python310\lib\site-packages\django\template\base.py", line 516, in parse
    compiled_result = compile_func(self, token)
  File "C:\Users\vivek\AppData\Local\Programs\Python\Python310\lib\site-packages\django\template\loader_tags.py", line 295, in do_extends
    nodelist = parser.parse()
  File "C:\Users\vivek\AppData\Local\Programs\Python\Python310\lib\site-packages\django\template\base.py", line 518, in parse
    raise self.error(token, e)
  File "C:\Users\vivek\AppData\Local\Programs\Python\Python310\lib\site-packages\django\template\base.py", line 516, in parse
    compiled_result = compile_func(self, token)
  File "C:\Users\vivek\AppData\Local\Programs\Python\Python310\lib\site-packages\django\template\loader_tags.py", line 234, in do_block
    nodelist = parser.parse(("endblock",))
  File "C:\Users\vivek\AppData\Local\Programs\Python\Python310\lib\site-packages\django\template\base.py", line 489, in parse
    raise self.error(token, e)
  File "C:\Users\vivek\AppData\Local\Programs\Python\Python310\lib\site-packages\django\template\base.py", line 487, in parse
    filter_expression = self.compile_filter(token.contents)
  File "C:\Users\vivek\AppData\Local\Programs\Python\Python310\lib\site-packages\django\template\base.py", line 605, in compile_filter
    return FilterExpression(token, self)
  File "C:\Users\vivek\AppData\Local\Programs\Python\Python310\lib\site-packages\django\template\base.py", line 706, in __init__
    raise TemplateSyntaxError(
django.template.exceptions.TemplateSyntaxError: Could not parse the remainder: ' {% if %} profile.profile_image else '{% static 'images/profile-face.png' %}' {% endif %' from 'profile.profile_image.url {% if %} profile.profile_image else '{% static 'images/profile-face.png' %}' {% endif %'
[ 2025-03-24 17:33:55,385 ] 248 django.request - ERROR - Internal Server Error: /advertisers/edit/
Traceback (most recent call last):
  File "C:\Users\vivek\AppData\Local\Programs\Python\Python310\lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
  File "C:\Users\vivek\AppData\Local\Programs\Python\Python310\lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "C:\Users\vivek\AppData\Local\Programs\Python\Python310\lib\site-packages\django\contrib\auth\decorators.py", line 60, in _view_wrapper
    return view_func(request, *args, **kwargs)
  File "C:\Users\vivek\Desktop\Projects\Cyber Guardia\Cyber-Bullying-Detection\advertisers\views.py", line 100, in edit_advertiser_profile
    return render(request, 'advertisers/adver_edit_profile.html', {'form': form})
  File "C:\Users\vivek\AppData\Local\Programs\Python\Python310\lib\site-packages\django\shortcuts.py", line 25, in render
    content = loader.render_to_string(template_name, context, request, using=using)
  File "C:\Users\vivek\AppData\Local\Programs\Python\Python310\lib\site-packages\django\template\loader.py", line 61, in render_to_string
    template = get_template(template_name, using=using)
  File "C:\Users\vivek\AppData\Local\Programs\Python\Python310\lib\site-packages\django\template\loader.py", line 15, in get_template
    return engine.get_template(template_name)
  File "C:\Users\vivek\AppData\Local\Programs\Python\Python310\lib\site-packages\django\template\backends\django.py", line 79, in get_template
    return Template(self.engine.get_template(template_name), self)
  File "C:\Users\vivek\AppData\Local\Programs\Python\Python310\lib\site-packages\django\template\engine.py", line 177, in get_template
    template, origin = self.find_template(template_name)
  File "C:\Users\vivek\AppData\Local\Programs\Python\Python310\lib\site-packages\django\template\engine.py", line 159, in find_template
    template = loader.get_template(name, skip=skip)
  File "C:\Users\vivek\AppData\Local\Programs\Python\Python310\lib\site-packages\django\template\loaders\cached.py", line 57, in get_template
    template = super().get_template(template_name, skip)
  File "C:\Users\vivek\AppData\Local\Programs\Python\Python310\lib\site-packages\django\template\loaders\base.py", line 28, in get_template
    return Template(
  File "C:\Users\vivek\AppData\Local\Programs\Python\Python310\lib\site-packages\django\template\base.py", line 154, in __init__
    self.nodelist = self.compile_nodelist()
  File "C:\Users\vivek\AppData\Local\Programs\Python\Python310\lib\site-packages\django\template\base.py", line 196, in compile_nodelist
    nodelist = parser.parse()
  File "C:\Users\vivek\AppData\Local\Programs\Python\Python310\lib\site-packages\django\template\base.py", line 518, in parse
    raise self.error(token, e)
  File "C:\Users\vivek\AppData\Local\Programs\Python\Python310\lib\site-packages\django\template\base.py", line 516, in parse
    compiled_result = compile_func(self, token)
  File "C:\Users\vivek\AppData\Local\Programs\Python\Python310\lib\site-packages\django\template\loader_tags.py", line 295, in do_extends
    nodelist = parser.parse()
  File "C:\Users\vivek\AppData\Local\Programs\Python\Python310\lib\site-packages\django\template\base.py", line 518, in parse
    raise self.error(token, e)
  File "C:\Users\vivek\AppData\Local\Programs\Python\Python310\lib\site-packages\django\template\base.py", line 516, in parse
    compiled_result = compile_func(self, token)
  File "C:\Users\vivek\AppData\Local\Programs\Python\Python310\lib\site-packages\django\template\loader_tags.py", line 234, in do_block
    nodelist = parser.parse(("endblock",))
  File "C:\Users\vivek\AppData\Local\Programs\Python\Python310\lib\site-packages\django\template\base.py", line 489, in parse
    raise self.error(token, e)
  File "C:\Users\vivek\AppData\Local\Programs\Python\Python310\lib\site-packages\django\template\base.py", line 487, in parse
    filter_expression = self.compile_filter(token.contents)
  File "C:\Users\vivek\AppData\Local\Programs\Python\Python310\lib\site-packages\django\template\base.py", line 605, in compile_filter
    return FilterExpression(token, self)
  File "C:\Users\vivek\AppData\Local\Programs\Python\Python310\lib\site-packages\django\template\base.py", line 706, in __init__
    raise TemplateSyntaxError(
django.template.exceptions.TemplateSyntaxError: Could not parse the remainder: ' if profile.profile_image else '/static/assets/Images/user_empty_profile.png'' from 'profile.profile_image.url if profile.profile_image else '/static/assets/Images/user_empty_profile.png''
[ 2025-03-24 17:36:05,605 ] 248 django.request - WARNING - Not Found: /advertisers/edit/font.css
[ 2025-03-24 17:36:07,141 ] 248 django.request - WARNING - Not Found: /advertisers/profile/usera/font.css
[ 2025-03-24 17:36:33,176 ] 248 django.request - WARNING - Not Found: /advertisers/profile/usera/font.css
[ 2025-03-24 17:36:34,153 ] 248 django.request - WARNING - Not Found: /advertisers/edit/font.css
[ 2025-03-24 17:39:39,483 ] 248 django.request - WARNING - Not Found: /advertisers/edit/font.css
[ 2025-03-24 17:39:46,078 ] 248 django.request - WARNING - Not Found: /advertisers/edit/font.css
[ 2025-03-24 17:39:47,779 ] 248 django.request - WARNING - Not Found: /advertisers/profile/usera/font.css
[ 2025-03-24 17:40:25,222 ] 248 django.request - WARNING - Not Found: /advertisers/edit/font.css
[ 2025-03-24 17:40:31,725 ] 248 django.request - ERROR - Internal Server Error: /advertisers/edit/
Traceback (most recent call last):
  File "C:\Users\vivek\AppData\Local\Programs\Python\Python310\lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
  File "C:\Users\vivek\AppData\Local\Programs\Python\Python310\lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "C:\Users\vivek\AppData\Local\Programs\Python\Python310\lib\site-packages\django\contrib\auth\decorators.py", line 60, in _view_wrapper
    return view_func(request, *args, **kwargs)
  File "C:\Users\vivek\Desktop\Projects\Cyber Guardia\Cyber-Bullying-Detection\advertisers\views.py", line 93, in edit_advertiser_profile
    return redirect('adver_profile', username=request.user.username)
  File "C:\Users\vivek\AppData\Local\Programs\Python\Python310\lib\site-packages\django\shortcuts.py", line 49, in redirect
    return redirect_class(resolve_url(to, *args, **kwargs))
  File "C:\Users\vivek\AppData\Local\Programs\Python\Python310\lib\site-packages\django\shortcuts.py", line 180, in resolve_url
    return reverse(to, args=args, kwargs=kwargs)
  File "C:\Users\vivek\AppData\Local\Programs\Python\Python310\lib\site-packages\django\urls\base.py", line 88, in reverse
    return resolver._reverse_with_prefix(view, prefix, *args, **kwargs)
  File "C:\Users\vivek\AppData\Local\Programs\Python\Python310\lib\site-packages\django\urls\resolvers.py", line 831, in _reverse_with_prefix
    raise NoReverseMatch(msg)
django.urls.exceptions.NoReverseMatch: Reverse for 'adver_profile' not found. 'adver_profile' is not a valid view function or pattern name.
[ 2025-03-24 17:40:48,553 ] 266 django.utils.autoreload - INFO - C:\Users\vivek\Desktop\Projects\Cyber Guardia\Cyber-Bullying-Detection\advertisers\views.py changed, reloading.
